retrofit2-clients/src/main/java/com/palantir/remoting3/retrofit2/AsyncSerializableErrorCallAdapterFactory.java

```diff
@@ -17,30 +15,51 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * Changes made:
- * - Made code style comply with Palantir checkstyle.
- * - Removed the ResponseCallAdapter
- * - Made the BodyCallAdapter create RemoteExceptions.
- * - Made the BodyCallAdapter tell the AsyncCallTag that this is an async call.
  */
 package com.palantir.remoting3.retrofit2;
 
-import com.palantir.remoting3.errors.SerializableErrorToExceptionConverter;
-import com.palantir.remoting3.okhttp.AsyncCallTag;
-import java.io.InputStream;
+import com.palantir.remoting3.okhttp.RemoteIoException;
+import java.io.IOException;
 import java.lang.annotation.Annotation;
 import java.lang.reflect.ParameterizedType;
 import java.lang.reflect.Type;
-import java.util.Collection;
 import java.util.concurrent.CompletableFuture;
 import retrofit2.Call;
 import retrofit2.CallAdapter;
 import retrofit2.Callback;
 import retrofit2.Response;
 import retrofit2.Retrofit;
 
-final class AsyncSerializableErrorCallAdapterFactory extends CallAdapter.Factory {
-    static final AsyncSerializableErrorCallAdapterFactory INSTANCE = new AsyncSerializableErrorCallAdapterFactory();
+/*
+ * Changes from {@link https://github.com/square/retrofit/blob/parent-2.3.0/retrofit-adapters/java8/src/main/java/retrofit2/adapter/java8/Java8CallAdapterFactory.java}
```

Comment:
you sure this is in the right place now?

Reply:
Haha yeah I moved it out of the license comment block because IDEs collapse that by default, so I never actually saw this link in the first place!